src/views/order_view.js

```diff
@@ -0,0 +1,55 @@
+import Backbone from 'backbone';
+import Quote from '../models/quote';
+// import QuoteView from '../views/quote_view';
+import Order from '../models/order';
+import OrderList from '../collections/order_list';
+import $ from 'jquery';
+
+
+const OrderView = Backbone.View.extend({
+  initialize(params) { //where do the params come from? app.js
+    this.template = params.template;
+    this.model = params.model;
+    this.listenTo(this.model, 'change', this.render);
+    //anytime the model changes, it will redraw it
+    this.listenTo(this.model.get('quote'), 'change', this.executeOrder);
+
+  },
+
+  executeOrder() {
+    console.log('execute order is being called');
+    let quote = this.model.get('quote');
+
+    if (this.model.get('buy')) {
+      if (this.model.get('targetPrice') >= quote.get('price')) {
+        quote.buy();
+        this.model.destroy();
+        this.remove();
+      }
+    } else {
+      if (this.model.get('targetPrice') <= quote.get('price')) {
+        quote.sell();
+        this.model.destroy();
+        this.remove();
+      }
+    }
```

Comment:
The last thing that needs to happen here is to add a trade to the trade history. Based on your code elsewhere, it looks like triggering a `trade` event on the bus would set things in motion properly.